portal-frontend/src/app/features/applications/application-details/roso-details/roso-details.component.html

```diff
@@ -23,21 +23,10 @@
 
   <div class="subheading2 grid-1">Project Duration</div>
```

Comment:
Should this be Removal of Soil Project Duration ? Why are we only updating some of the references?

Reply:
My bad for missing that, updated!